packages/slate-react/src/components/leaf.js

```diff
@@ -5,6 +5,10 @@ import SlateTypes from 'slate-prop-types'
 
 import OffsetKey from '../utils/offset-key'
 
+const attributes = {
+  'data-text': true,
+}
```

Comment:
Please see comment below to remove this.

Reply:
Fixed~